jaeger-core/src/test/java/io/jaegertracing/internal/samplers/PerOperationSamplerTest.java

```diff
@@ -169,10 +169,29 @@ public void testUpdateAddOperation() {
                                                      DEFAULT_LOWER_BOUND_TRACES_PER_SECOND,
                                                      parametersList));
 
-    assertEquals(1, operationToSamplers.size());
+    assertEquals(1, undertest.getOperationNameToSampler().size());
     assertEquals(new GuaranteedThroughputSampler(SAMPLING_RATE,
                                                  DEFAULT_LOWER_BOUND_TRACES_PER_SECOND),
-                 operationToSamplers.get(OPERATION));
+            undertest.getOperationNameToSampler().get(OPERATION));
   }
 
+  @Test
+  public void testAbsentOperationIsRemoved() {
+    String absentOp = "ShouldBeRemoved";
+    operationToSamplers.put(absentOp, mock(GuaranteedThroughputSampler.class));
+    PerOperationSamplingParameters perOperationSamplingParameters1 =
+            new PerOperationSamplingParameters(OPERATION, new ProbabilisticSamplingStrategy(SAMPLING_RATE));
```

Comment:
Shouldn't `operationToSamplers` be passed in here to control the starting state of the test?

Reply:
undertest is inited with the operationToSamplers during setup, adding a sampler to the map should do the job regarding the starting state.